modules/update-upgrading-oc-adm-upgrade-status.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-status_{context}"]
+= Retrieving information about a cluster update using oc adm upgrade status (Technology Preview)
+
+When updating your cluster, it is useful to understand how your update is progressing. While the `oc adm upgrade` command returns limited information about the status of your update, this release introduces the `oc adm upgrade status` command as a Technology Preview feature. This command decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
+
+The `oc adm upgrade status` command is read-only and will never alter any state in your cluster. 
+
+:FeatureName: The `oc adm upgrade status` command
+include::snippets/technology-preview.adoc[]
+
+The `oc adm upgrade status` command can be used for clusters from version 4.12 up to the latest supported release. 
+
+[IMPORTANT]
+====
+While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `OC_ENABLE_CMD_UPGRADE_STATUS` Technology Preview environment variable, otherwise the {oc-first} will not recognize the command and you will not be able to use the feature. 
+====
+
+.Procedure 
+
+. Set the `OC_ENABLE_CMD_UPGRADE_STATUS` environmental variable to `true` by running the following command:
++
+[source,terminal]
+----
+$ export OC_ENABLE_CMD_UPGRADE_STATUS=true
+----
++
+. Run the `oc adm upgrade status` command:
++
+[source,terminal]
+----
+$ oc adm upgrade status
+----
++
+The following output is an example of an update that is progressing successfully:
++
+.Example output
```

Comment:
Just a suggestion, but if you wanna trim a line, could rework the sentence into the block title:
```suggestion
.Example output for an update progressing successfully
```